content/docs/operating/security.md

```diff
@@ -112,6 +113,28 @@ Prometheus and its components do not provide any server-side
 authentication, authorisation or encryption. If you require this, it is
 recommended to use a reverse proxy.
 
+As administrative and mutating endpoints are intended to be accessed via simple
+tools such as cURL, there is no built in
+[CSRF](https://en.wikipedia.org/wiki/Cross-site_request_forgery) protection as
+that would break such use cases. Accordingly when using a reverse proxy, you
+may wish to block such paths to prevent CSRF.
+
+For non-mutating endpoints, you may wish to set [CORS
+headers](https://fetch.spec.whatwg.org/#http-cors-protocol) such as
+`Access-Control-Allow-Origin` in your reverse proxy to prevent
+[XSS](https://en.wikipedia.org/wiki/Cross-site_scripting).
+
+If you are composing PromQL queries that include input from untrusted users
+(e.g. URL paramaters to console templates, or something you built yourself) who
+are not meant to be able to run aribtrary PromQL queries make sure any
+untrusted input is appropriately escaped to prevent injection attacks. For
+example `up{job="<user_input>"}` could become `up{job=""} or
+some_metric{zzz=""}`.
```

Comment:
Maybe add "if <user_input> is `"} or some_metric{zzz="` - a surprising amount of people find this non-obvious.